docs/docs/data-sources/Overview.md

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 1
+---
+
+## Introduction to Data Sources
+
+In this section, we'll explore the concept of data sources within ALGOMO. Data sources play a crucial role in providing information for bots to answer questions. Think of them as the building blocks of knowledge that make up your bot. These sources can take various forms, such as PDF documents, structured Question and Answers (intents), website searches (via Bing or index versions), and more.
```

Comment:
I prefer `Algomo` rather than `ALGOMO`.

I think the `(via Bing or index versions)` should be changed to `(via Bing or scraped version)` to match the terminology in the app.

Reply:
Not my problem